secret_hitler/app.py

```diff
@@ -1,8 +1,31 @@
 import os
 import discord
-from discord.ext import commands
+import logging
 from dotenv import load_dotenv
 from secret_hitler.game import Game, GameStates
+from discord.ext import commands
+from secret_hitler.game import Game, GameStates, Player
+from secret_hitler import config
+
+logFormatter = logging.Formatter('[%(asctime)s][%(levelname)s][%(name)s] %(message)s')
+
+# log discord API messages to discord.log file
+discordLogger = logging.getLogger('discord')
+discordLogger.setLevel(logging.DEBUG)
+handler = logging.FileHandler(filename='discord.log', encoding='utf-8', mode='w')
+handler.setFormatter(logFormatter)
+discordLogger.addHandler(handler)
+
+# logger for this application
+logger = logging.getLogger("secret_hitler")
+logger.setLevel(logging.DEBUG)
+handler = logging.StreamHandler()
+handler.setFormatter(logFormatter)
+logger.addHandler(handler)
+
+#constants for ja or nein voting
+JA='sh_ja'
+NEIN='sh_nein'
```

Comment:
That's a good idea to make the names of these emotes more distinct. Should we got a bit further and spell out "secret_hitler"? "sh" is short and still may collide. Being more specific here might be good.

```suggestion
JA='secret_hitler_ja'
NEIN='secret_hitler_nein'
```

Reply:
fair point.  I'm going to do this in a new PR though because there's another place it would need to change (config.py) and also I wasn't happy that I needed the same constant in two places like that, so I was considering reworking that anyway